dashboard/app/views/pd/workshop_enrollment/_enroll_form.html.haml

```diff
@@ -0,0 +1,50 @@
+- if @enrollment.errors.any?
+  #error_explanation
+    %h2= "#{pluralize(@enrollment.errors.count, 'error')}:"
+    %ul
+      - @enrollment.errors.each do |name, message|
+        - if name == :school_district
+          %li= "School district or school ZIP is required"
+        - else
+          %li= @enrollment.errors.full_message(name, message)
+
+=form_for @enrollment, url: {action: action} do |form|
+  %div.control-group
+    %label
+      Your Name
+      %span.form-required-field
+        *
+    %div.controls
+      =form.text_field :name, class: 'span5'
+  %div.control-group
+    %label
+      Email Address
+      %span.form-required-field
+        *
+    %div.controls
+      - readonly_email = current_user && !current_user.email.blank? && !@enrollment.email.blank?
+      =form.text_field :email, class: 'span5', readonly: readonly_email,
+        title: readonly_email ? 'Email can be changed in account settings' : nil
+  - unless current_user
+    %div.control-group
+      %label
+        Confirm Email Address
+        %span.form-required-field
+          *
+      %div.controls
+        =form.text_field :email_confirmation, class: 'span5'
+  %div.control-group
+    %label
+      District
+    - error_class = (@enrollment.errors.include? :school_district) ? "field_with_errors_div" : ""
+    #school_district.span5{class: error_class, style: "border: solid 1px #bbb; border-radius: 4px; padding: 15px; margin-left: 0px; box-sizing: border-box;"}
```

Comment:
I'm leaving this alone for now. We can revisit this css when we address the upcoming district explorer changes.